remote-trait-object-macro/src/service.rs

```diff
@@ -121,12 +121,12 @@ pub fn service(args: TokenStream2, input: TokenStream2) -> Result<TokenStream2,
 
     let id = id::generate_id(&source_trait, &args)?;
     let dispatcher = dispatcher::generate_dispatcher(&source_trait, &args)?;
-    let remote = remote::generate_remote(&source_trait, &args)?;
+    let stub = proxy::generate_proxy(&source_trait, &args)?;
```

Comment:
How the `stub` and the `proxy` are different?
If they indicate the same concept, let's use only one name for the concept.

Reply:
Same. I updated to use only `proxy`.